static/bidder-info/cwire.yaml

```diff
@@ -0,0 +1,10 @@
+endpoint: "https://ortb.cwi.re/v1/bid"
+maintainer:
+  email: "mateja.maric@cwire.com"

```

Comment:
Please use a group email here. We strongly discourage using individual emails because there might be situations where a person say moves to a diff company, we would have no way to reach them.

Reply:
Fixed